packages/by-name/kata/kata-runtime/0017-Include-reference-in-logs-when-auth-failure.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: jmxnzo <jakob.lammering@ruhr-uni-bochum.de>
+Date: Mon, 2 Dec 2024 09:45:46 +0100
+Subject: [PATCH] Include reference in logs when auth failure
+
+---
+ src/tools/genpolicy/src/registry.rs | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/tools/genpolicy/src/registry.rs b/src/tools/genpolicy/src/registry.rs
+index 9491b2acfb8af1f8c4e4c971a6787c2d803e46ec..5815cff89e4ddcfe94f0bc2c7d44263efda20311 100644
+--- a/src/tools/genpolicy/src/registry.rs
++++ b/src/tools/genpolicy/src/registry.rs
+@@ -159,7 +159,7 @@ impl Container {
+                 })
+             }
+             Err(oci_client::errors::OciDistributionError::AuthenticationFailure(message)) => {
+-                panic!("Container image registry authentication failure ({}). Are docker credentials set-up for current user?", &message);
++                panic!("Container image registry authentication failure ({}) for {:#?}. Are docker credentials set-up for current user?", &message, &reference);
```

Comment:
`{:#?}` is probably a debug print that shows the internal structure of https://github.com/youki-dev/oci-spec-rs/blob/e1dded7b5ff2d448d8802e2fe9202c86eb6f6c30/src/distribution/reference.rs#L77. Can we get a readable image reference string here?